src/o3logon_fmt_plug.c

```diff
@@ -41,7 +41,7 @@ john_register_one(&fmt_o3logon);
 #define ALGORITHM_NAME                  "SHA1 DES 32/" ARCH_BITS_STR
 
 #define BENCHMARK_COMMENT               ""
-#define BENCHMARK_LENGTH                0x107
+#define BENCHMARK_LENGTH                7
```

Comment:
OK, but I am puzzled why "Many salts" gets limited to as few as 3 salts when the speed is rather high (about as high as for o10glogon above, where we got ~150 salts in a similar test). I guess very different MKPC*OMP_SCALE, but why would optimal be very different for formats that are of similar speed? Confirmed: o10glogon has `16*8`, whereas o3logon has `64*128`. One or both of these are probably mistuned.

```
[solar@localhost run]$ GOMP_CPU_AFFINITY=0-255 GOMP_SPINCOUNT=10000 ../run/john -test -form=o3logon  
Will run 256 OpenMP threads
Benchmarking: o3logon, Oracle O3LOGON protocol [SHA1 DES 32/64]... (256xOMP) DONE
Warning: "Many salts" test limited: 3/256
Many salts:     4279K c/s real, 17453 c/s virtual
Only one salt:  2150K c/s real, 17568 c/s virtual

[solar@localhost run]$ GOMP_CPU_AFFINITY=0-255 GOMP_SPINCOUNT=10000 ../run/john -test -form=o3logon
Will run 256 OpenMP threads
Benchmarking: o3logon, Oracle O3LOGON protocol [SHA1 DES 32/64]... (256xOMP) DONE
Warning: "Many salts" test limited: 3/256
Many salts:     4309K c/s real, 17463 c/s virtual
Only one salt:  2129K c/s real, 17557 c/s virtual
```